app/src/main/java/edu/uc/groupProject/topten/MainActivity.kt

```diff
@@ -32,20 +32,6 @@ class MainActivity : AppCompatActivity() {
                 .commitNow()
         }
 
-        /* val textView = findViewById<TextView>(R.id.textview)
-        textView.setText("")
-
-        mvm.fetchFirestoreList()
-
-        mvm.list.observeForever {
-            it.forEach {
-                textView.append(it.title)
-                textView.append("\n")
-            }
-        }
-
-        */
-
```

Comment:
Best to use version control to get rid of unused code instead of comments

Reply:
Yes.

 Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.